static/openrpc.json

```diff
@@ -1117,6 +1132,39 @@
                 }
               }
             },
+            "stateChanges": {
+              "type": "array",
+              "description": "(optional) - On successful simulation of `InvokeHostFunction` operations, this field will be an array of `LedgerEntry`s before and after simulation occurred. Note that _at least_ one of `before` or `after` will be present: `before` and no `after` indicates a deletion event, the inverse is a creation event, and both present indicates an update event. Or just check the `type`.",
+              "items": {
+                "type": "object",
+                "required": [
+                  "type",
+                  "key"
+                ],
+                "properties": {
+                  "^type$": {
+                    "oneOf": [
+                      {},
+                      {},
+                      {}
+                    ],
+                    "description": "Indicates if the entry was created (1), updated (2), or deleted (3)"
+                  },
+                  "key": {
+                    "type": "string",
+                    "description": "Base64 - the `LedgerEntryKey` for this delta"
```

Comment:
Question: Is it LedgerKey or LedgerEntryKey? could not find LedgerEntryKey in `xdr_generated`.go so asking to confirm

Reply:
Ah yeah, typo oops